### PR TITLE
Oskar/notification_messages

### PR DESCRIPTION
**Pull request by OskarAhl**
* Update Toast to Notification messages
* Add missing notification messages

### CODE REVIEW

File: src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    const onClick = () => destroy(true);
+
+    if (data.is_auto_close) {
+        setTimeout(destroy, data.delay || default_delay);
+    }
+
+    return (
+        <div className={
+            classNames('notification', types[data.type], {
+                'notification--small': data.size === 'small',
```
Comment (zakibinary) [inline src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx]: Can you add `parentheses` around the condition?

File: src/javascript/app_2/Stores/Helpers/client-notifications.js [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                replacers={{ '1_2': <a className='link link--white' target='_blank' href={urlFor('user/settings/detailsws', undefined, undefined, true)} /> }}
+            />
+        ),
+        type: 'danger',
+    },
+};
+
+const hasMissingRequiredField = (State, client) => {
+    const { landing_company_shortcode } = client;
+    const is_svg = landing_company_shortcode === 'svg' || landing_company_shortcode === 'costarica';
```
Comment (zakibinary) [inline src/javascript/app_2/Stores/Helpers/client-notifications.js]: Same here. `parentheses` around the conditions.

File: src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import classNames          from 'classnames';
+import { Icon }            from 'Assets/Common';
+import { IconDanger }      from 'Assets/Common/icon-danger.jsx';
+import { IconInformation } from 'Assets/Common/icon-information.jsx';
+import { IconWarning }     from 'Assets/Common/icon-warning.jsx';
+import { IconSuccess }     from 'Assets/Common/icon-success.jsx';
+import CloseButton         from './close-button.jsx';
+import {
+    default_delay,
+    types }             from './constants';
```
Comment (cakasuma) [inline src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx]: nit, space here

File: src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+const Notification = ({
+    data,
+    removeNotification,
+}) => {
+    const destroy = (is_closed_by_user) => {
+        removeNotification(data);
+
+        if (typeof data.closeOnClick === 'function') {
```
Comment (cakasuma) [inline src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx]: is this checking closeOnClick exists?, i thought `if (data.closeOnClick)` is also fine? or is there other case that data.closeOnClick is not a function?

File: src/javascript/app_2/Stores/client-store.js [hunk 5]
```diff
@@ -17,6 +17,7 @@ import {
     State }                    from '_common/storage';
 import BaseStore               from './base-store';
 import { buildCurrenciesList } from './Modules/Trading/Helpers/currency';
```
Comment (cakasuma) [inline src/javascript/app_2/Stores/client-store.js]: nit, space here

File: src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx (hunk 3 above)
Comment (cakasuma) [inline src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx]: nit, space here

File: src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx (hunk 4 above)
Comment (cakasuma) [inline src/javascript/app_2/App/Components/Elements/NotificationMessage/notification.jsx]: is there any case that data.closeOnClick is not a function. is this checking if data.closeOnClick is exists?

**Pull request closed without merging**